src/components/Map/Controls/index.tsx

```diff
@@ -116,6 +117,29 @@ export const MapControls = () => {
         </Control>
         <Control position="bottomleft">
           <Stack display={"flex"} direction={"column"} rowGap={1}>
+            <Box
+              sx={{
+                padding: {
+                  xs: "0px 0px 28px 8px",
+                  sm: "0",
+                  md: "0",
+                },
+              }}
+            >
+              <Link href="https://afet.org">
```

Comment:
How about we open the off-site redirects in a new tab?

Reply:
fixed